src/status_im2/subs/communities.cljs

```diff
@@ -116,15 +116,16 @@
 (re-frame/reg-sub
  :communities/community-ids-by-user-involvement
  :<- [:communities/communities]
+ :<- [:communities/my-pending-requests-to-join]
  ;; Return communities splitted by level of user participation. Some communities user
  ;; already joined, to some of them join request sent and others were opened one day
  ;; and their data remained in app-db.
  ;; Result map has form: {:joined [id1, id2] :pending [id3, id5] :opened [id4]}"
- (fn [communities]
+ (fn [[communities requests]]
    (reduce (fn [acc community]
              (let [joined?    (:joined community)
-                   requested? (pos? (:requested-to-join-at community))
```

Comment:
we get active requests from the requests list instead now, rather than the `community` object. Discussed this with @borismelnik 👍